### ISSUE TITLE
Game is deserted

### ISSUE THREAD

**Issue by PavanLuca** [browserquest/BrowserQuest#223]
Every time I open the game there's no one, neither players nor AI. I suppose there should be those enemies that I read on achievements section even if there's no player, at least I'd fight AI instead of leaving.

**Comment by Veradictus** [comment]
Yeah, BrowserQuest has died a long time ago. I've continued development with my own version if you're interested - https://github.com/Kaetram/Kaetram-Open

Game is set to launch officially October 7th.

**Comment by PavanLuca** [comment]
Seems great. And do you plan to release it as APT, flat or snap, too?
